nh_clinical/tests/nh_clinical_patient_allocation/test_allocation_complete.py

```diff
@@ -0,0 +1,61 @@
+from openerp.tests.common import SavepointCase
+
+
+class TestAllocationComplete(SavepointCase):
+
```

Comment:
As discussed have not added this because the class name adequately conveys the necessary information which is that this class is for testing the allocation model's `complete` method.